pkg/types/status.go

```diff
@@ -20,18 +23,21 @@ var (
 		"will_not_fix",
 		"fix_deferred",
 		"end_of_life",
+		"needed",
+		"pending",
 	}
 )
 
 const (
 	StatusUnknown Status = iota
 	StatusNotAffected
 	StatusAffected
-	StatusFixed
+	StatusFixed // Ubuntu "Released" Mapped to Fixed
 	StatusUnderInvestigation
-	StatusWillNotFix // Red Hat specific
+	StatusWillNotFix // Red Hat specific // Ubuntu Specific "ignored" Superseded || End of life || End of Kernal support
```

Comment:
```suggestion
	StatusWillNotFix
```
I think that now that this is not specific to a single provider, it makes sense to just remove the comment.